modules/juce_graphics/geometry/juce_Point.h

```diff
@@ -41,32 +45,32 @@ class Point
 {
 public:
     /** Creates a point at the origin */
-    Point() noexcept : x(), y() {}
+    constexpr Point() noexcept : x(), y() {}

```

Comment:
The macro erases constexpr keyword if using VS2013 or 2015. That's why it's there. Microsoft uses the macro themselves for this purpose.